firestore/client/version_info.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_FIRESTORE_CLIENT_VERSION_INFO_H_
+#define GOOGLE_CLOUD_CPP_FIRESTORE_CLIENT_VERSION_INFO_H_
+
+#define FIRESTORE_CLIENT_VERSION_MAJOR 
```

Comment:
Typically `version_info.h` is generated from a `version_info.h.in` because we wanted to keep the version numbers in just one place.

Reply:
got it.